lib/master_manipulator/version.rb

```diff
@@ -1,5 +1,5 @@
 module MasterManipulator
   module Version
-    STRING = '1.2.3'
+    STRING = '1.2.4'
```

Comment:
This might be a breaking change that warrants 2.0.0. Many modules projects depend on ```restart_puppet_server``` to have a certain signature and behave a certain way.